test/lib/ufe/testUIIcons.py

```diff
@@ -115,7 +115,6 @@ def testUIIcons(self):
             ('NurbsCurves',             'out_USD_UsdGeomCurves.png'),
             ('NurbsPatch',              'out_USD_NurbsPatch.png'),
             ('OpenVDBAsset',            'out_USD_UsdGeomXformable.png'),
-            ('PackedJointAnimation',    'out_USD_SkelAnimation.png'),
```

Comment:
@dj-mcg What do we do about this prim type still appearing in the "Add New Prim" menu? This UI test is ensuring that each prim gets the proper icon from Ufe.
![PackedJointAnimation](https://github.com/Autodesk/maya-usd/assets/23455376/d1199f3d-0060-49fc-a209-1ce58da518c2)

Reply:
This test was still passing for me locally even with the prim removed from USD. I didn't dig too deeply into why. I added versioning with this last commit to make sure everything is above board - please review and let me know if this addresses concerns, thanks!

Reply:
Sorry just to confirm. With USD v23.11 there will no longer be a PackedJointAnimation prim type? So that item won't appear in our "Add new prim" menu when using 23.11.

Reply:
That is correct. What I find a little surprising is that both tests pass as is (ie. still referencing the PackedJointAnimation prim) even when built/run against 23.11, so they are either both tolerant of missing prim types or not catching the errors if they are supposed to

Reply:
I still see the PackedJointAnimation in the USD dev branch https://github.com/PixarAnimationStudios/OpenUSD/blob/dev/pxr/usd/usdSkel/schema.usda#L144 - it has only been deprecated, but not removed yet.

Reply:
That will be gone with the next push to USD dev, likely at the end of this week